src/testsuite.cpp

```diff
@@ -275,14 +275,16 @@ int compile(const submission& target, int boxid, int spBoxid) {
   } else if (target.lang == "python2") {
     args = {"/usr/bin/env", "python2", "-m", "py_compile", "main.py"};
   } else if (target.lang == "python3") {
+    args = {"/usr/bin/env", "python3.7", "-c",
```

Comment:
I think here is a mistake when merging commits.

Reply:
Fixed at 0ecef8c